cfme/tests/configure/test_visual_cloud.py

```diff
@@ -19,13 +19,6 @@
               pytest.mark.usefixtures("openstack_provider")]
 
 # TODO When all of these classes have widgets and views use them in the tests
```

Comment:
Required: Please remove if it is already done, if not, then also move to list of classes